types/types.proto

```diff
@@ -61,8 +61,9 @@ message RequestQuery {
 message RequestBeginBlock {
   bytes hash = 1;
   Header header = 2 [(gogoproto.nullable)=false];
-  repeated bytes absent_validators = 3;
-  repeated Evidence byzantine_validators = 4 [(gogoproto.nullable)=false];
+  repeated RequestValidator present_validators = 3;
```

Comment:
we can just pass one list of validators here, where each has a bool that says whether they signed or not. and we don't need the index since they'll be in order, but might as well still have pubkey, address, power